packages/features/ee/sso/lib/saml.ts

```diff
@@ -31,7 +31,7 @@ export const isSAMLAdmin = (email: string) => {
 };
 
 export const samlTenantProduct = async (prisma: PrismaClient, email: string) => {
-  const user = await prisma.user.findUnique({
+  const user = await prisma.user.findFirst({
     where: {
       email,
```

Comment:
I am not sure how the org context would reach here but there has to be one here I guess.